docs/sop.rst

```diff
@@ -0,0 +1,135 @@
+Operating Clowder
+=================
+
+Two primary aspects of operating Clowder: Operating the apps managed by Clowder and operating
+Clowder itself.
+
+Contrary to many other deployment models where apps configure themselves, an application that is
```

Comment:
"configure themselves" sounds odd.  I wonder if saying something more like "where devs are burdened with the repsponsibility of wiring up all aspects of their applications in k8s", though that is much more wordy.

Reply:
Agreed